spec/features/routing/routing_tags/destroy_with_linked_rate_management_spec.rb

```diff
@@ -0,0 +1,151 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe 'Delete Routing Tag with linked rate management', type: :feature, js: true do
```

Comment:
move as context to `spec/features/routing/routing_tags/destroy_routing_tag_spec.rb`